MandelaK/SendIT-API
# #[161862208] setup travis and code climate

#### What does this PR do?
We set up our travis, code climate and coveralls integration. This should enable us continuously monitor our code quality and stick to standards required for completing this challenge
 
#### Description of Task to be completed?
We are supposed to use various continuous integration and code quality tools. This is to ensure we continuously monitor our commit history and ensure we don't accidentally add bugs to our app.

#### How should this be manually tested?
This can be manually tested by making commits and pull requests. With each of these, travis should run tests on our builds and tell us the results. This is useful for ensuring we only commit or merge passing code.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?
#161862208
